gprofiler/metadata/system_metadata.py

```diff
@@ -132,7 +134,37 @@ def get_mac_address() -> str:
         address = ":".join(["%02X" % i for i in mac])
         return address
 
-    return "unknown"
+    return UNKNOWN_VALUE
+
+
+def get_cpu_info() -> Tuple[str, str]:
+    """
+    Parse /proc/cpuinfo to get model name & flags.
+    """
+    try:
```

Comment:
I would want this in granulate-utils , but okay for now

Reply:
Not now